pages/contracts/_meta.json

```diff
@@ -2,6 +2,7 @@
   "overview": "Overview",
   "deployment-addresses": "Deployment Addresses",
   "audits": "Audits",
+  "potential-errors": "Potential Errors",
```

Comment:
it sounds like unexpected errors :) better to rename it to "Errors Handling"

Reply:
Renamed